logic/src/commonTest/kotlin/com/wire/kalium/logic/sync/PendingEventsBufferTest.kt

```diff
@@ -0,0 +1,95 @@
+package com.wire.kalium.logic.sync
+
+import com.wire.kalium.logic.framework.TestEvent
+import kotlinx.coroutines.test.runTest
+import kotlin.test.BeforeTest
+import kotlin.test.Test
+import kotlin.test.assertFalse
+import kotlin.test.assertTrue
+
+class PendingEventsBufferTest {
+
+    private lateinit var eventsBuffer: PendingEventsBuffer
+
+    @BeforeTest
+    fun setup() {
+        eventsBuffer = PendingEventsBuffer()
+    }
+
+    @Test
+    fun givenAnAddedEvent_whenCheckingIfPresent_thenShouldReturnTrue() = runTest {
+        val event = TestEvent.memberJoin("testEvent")
+        eventsBuffer.addToBuffer(event)
+
+        val result = eventsBuffer.isEventPresentInBuffer(event)
+
+        assertTrue(result)
+    }
+
+    @Test
+    fun givenAnEventThatWasNotAdded_whenCheckingIfPresent_thenShouldReturnFalse() = runTest {
+        val event = TestEvent.memberJoin("testEvent")
+
+        val result = eventsBuffer.isEventPresentInBuffer(event)
+
+        assertFalse(result)
+    }
+
+
+    @Test
+    fun givenAnAddedEvent_whenRemovingIt_thenShouldReturnTrue() = runTest {
+        val event = TestEvent.memberJoin("testEvent")
+        eventsBuffer.addToBuffer(event)
+
+        val result = eventsBuffer.removeEventFromBuffer(event)
+
+        assertTrue(result)
```

Comment:
I suggest to check wether the event is present as well.

Reply:
Ooh yeah, I forgot those.

I'll add tests just for this.